src/compiler/types.ts

```diff
@@ -1619,6 +1619,7 @@ namespace ts {
         getTypeReferenceSerializationKind(typeName: EntityName): TypeReferenceSerializationKind;
         isOptionalParameter(node: ParameterDeclaration): boolean;
         isArgumentsLocalBinding(node: Identifier): boolean;
+        getSymbolAtLocation(node: Node): Symbol;

```

Comment:
see comment for `lookupSpecifierName` - I think that checker should expose more concrete function like `getModuleRelativePath` or `getModuleNameAsRelativePath`